tests/integration/integration_test.go

```diff
@@ -255,20 +255,21 @@ func runPolicyPackIntegrationTest(
 	// Cleanup already registered via defer.
 }
 
-// Test invalid policies.
-func TestInvalidPolicy(t *testing.T) {
-	runPolicyPackIntegrationTest(t, "invalid_policy", NodeJS, nil, []policyTestScenario{
-		{
-			WantErrors: []string{`Invalid policy name "all". "all" is a reserved name.`},
-		},
-		{
-			WantErrors: []string{`enforcementLevel cannot be explicitly specified in properties.`},
-		},
-		{
-			WantErrors: []string{`"enforcementLevel" cannot be specified in required.`},
-		},
-	})
-}
+//
```

Comment:
All the temporary commented-out tests need to be uncommented.

Reply:
hah thanks for catching that

Reply:
Looks like this one test is still commented out.

Reply:
🤦